modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,180 @@
+
+.OVS `balance-slb` mode ` operating on a localnet with two NADs
+image::552_OpenShift_slb_mode_0625.png[OVS `balance-slb` mode ` operating on a localnet with two NADs]
+
+You can integrate the `balance-slb` mode interface into primary or secondary network types by using OVS bonding. Note the following points about OVS bonding:
+
+* Supports the OVN-Kubernetes CNI plugin and easily integrates with the plugin.
+* Natively supports `balance-slb` mode. 
+
+.Prerequisites
+
+* You have more than one physical interface attached to your primary network and you defined the interfaces in a `MachineConfig` file.
+* You created a manifest object and defined a customized `br-ex` bridge in the object configuration file.
+* You have more than one physical interfaces attached to your primary network and you defined the interfaces in a NAD file.
+
+.Procedure
+
+. For each bare-metal host that exists in a cluster, in the `install-config.yaml` file for your cluster define a `networkConfig` section similar to the following example:
++
+[source,yaml]
+----
+# ...
+networkConfig:
+  interfaces:
+    - name: enp1s0 <1>
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp2s0 <2>
+      type: interface
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        enabled: false
+    - name: enp3s0 <3>
+      type: interface
+      state: up
+      ipv4:
+        enabled: false
+      ipv6:
+        enabled: false
+# ...
+----
+<1> The interface for the provisioned network interface controller (NIC). 
+<2> The first bonded interface that pulls in the Ignition config file for the bond interface.
+<3> The second bonded interface is part of a minimal configuration that pulls ignition during cluster installation.
+
+. Define each network interface in an NMState configuration file:
++
+.Example NMState configuration file that defines many network interfaces
+[source,yaml]
+----
+ovn:
+  bridge-mappings:
+    - localnet: localnet-network
+      bridge: br-ex
+      state: present
+interfaces:
+  - name: br-ex
+    type: ovs-bridge
+    state: up
+    ipv4:
+      enabled: false
+      dhcp: false
+    ipv6:
+      enabled: false
+      dhcp: false
```

Comment:
We removed this IP disable in customer config.   The `type: ovs-bridge`   itself doesn't get an IP, only the `ovs-interface` type.